### PR TITLE
Add RAG documentation

### PR DESCRIPTION
**Pull request by lesters**
@kkraune Please review
@bratseth FYI

### CODE REVIEW

**Review by bratseth — APPROVED** [review]
LGTM!

File: en/llms-in-vespa.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
+---
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Large Language Models in Vespa"
+redirect_from:
+- /documentation/llms-in-vespa.html
+---
+
+Large Language Models (LLMs) are AI systems that generate human-like text,
```
[the remark this answers is not in the document]
Comment (lesters) [reply en/llms-in-vespa.md:8]: Good point, I'll add that

File: en/llms-rag.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    --header="X-LLM-API-KEY:..." \ 
+    query="what was the manhattan project?" \
+    searchChain=rag \
+    format=sse 
+```
+
+However, notice here the use of the `query` query parameter. In [LLMs in
+Vespa](llms-in-vespa.html), we used a `prompt` parameter to set up the prompt
+to send to the LLM. You can also do that in the `RAGSearcher`, however this means 
+that no actualy query is run in Vespa. For Vespa to run a search, you need to
```
Comment (kkraune) [inline en/llms-rag.md:67]:
```suggestion
that no actual query is run in Vespa. For Vespa to run a search, you need to 
```

**Review by kkraune — APPROVED** [review]
Great! A few things:
* A nice illustration will make it better, like https://blog.vespa.ai/assets/2023-09-08-search-vespa-ai/vespa-search-components.svg in the blog post (but with less details).
* json and xml highlighting can be used for readability

The guides are well written - must be added to _data/sidebar.yml, too

**Pull request merged**